cfme/tests/infrastructure/test_snapshot.py

```diff
@@ -499,3 +499,454 @@ def test_create_snapshot_via_ae(appliance, request, domain, small_test_vm):
 
     # Clean up if it appeared
     snapshot.delete()
+
+
+@pytest.mark.manual
+@test_requirements.snapshot
+@pytest.mark.tier(1)
+def test_rhos_notification_for_snapshot_failures():
```

Comment:
I think you need a provider marker here.

The module is marked for RHV and vmware, you will need something like:
`@pytest.mark.provider([Openstack], override=True)`

Above all that - this is `cfme/tests/infrastructure/test_snapshot.py`, and I think this test function belongs in `cfme/tests/cloud/<new_module>` since its not for an infrastructure provider.

Reply:
Moved to /cloud/test_snapshot.